engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/content-host-details.controller.js

```diff
@@ -24,6 +24,10 @@ angular.module('Bastion.content-hosts').controller('ContentHostDetailsController
         $scope.getHostStatusIcon = ContentHostsHelper.getHostStatusIcon;
         $scope.getHostPurposeStatusIcon = ContentHostsHelper.getHostPurposeStatusIcon;
 
+        if (!$scope.organization) {
```

Comment:
I'm going to test it out - but can you think of any reason to not remove the if statement around this assignment? I don't think it's needed (it hung around from a previous commit)